### ISSUE TITLE
Update Copyright Page

### ISSUE THREAD

**Issue by mikelmaron** [openstreetmap/openstreetmap-website#5132]
https://www.openstreetmap.org/copyright is the second most visited page on the website, but is not a great place to land and understand the project. It needs to present legal information, but should also explain the project well and direct people to get involved (https://www.openstreetmap.org/about does a decent job).

The CWG had developed [recommendations on changes to content and presentation](https://docs.google.com/document/d/1qthlWHa3-cQ9KcHai-TORAEnHD3LBI0vTi2OJ1TMz_k/edit), and that text has been ok'd by the LWG and Board.

This now needs implementation. It reasonably straightforward. Requires some changes to layout in [the view](https://github.com/openstreetmap/openstreetmap-website/blob/master/app/views/site/copyright.html.erb), updates/additions to the [translation file](https://github.com/openstreetmap/openstreetmap-website/blob/cee9818dfc9ac3f6eae01cdb51df9093ae5d1322/config/locales/en.yml#L2004), and perhaps facilitation of sufficient number of translations in the translation system.

Would be good to have guidance from maintainers if this is simple enough that it is sufficient to make this change in those text files and submit as a PR, or require someone to get the website running locally to test? Also, what is sufficient translation coverage for new strings in order to release an update?

And if someone wants to jump in and help with this small task would be much appreciated!

**Comment by mikelmaron** [comment]
Note there's a few other old issues which could possible be swept into these updates as well

* https://github.com/openstreetmap/openstreetmap-website/issues/3998
* https://github.com/openstreetmap/openstreetmap-website/issues/2077
* https://github.com/openstreetmap/openstreetmap-website/issues/1878

**Comment by gravitystorm** [comment]
> Also, what is sufficient translation coverage for new strings in order to release an update?

We make the changes to en.yml, the PR is merged, and then the translations come in at their own pace. We receive updates from translatewiki a few times per week. Deployments also happen every few days. But there's no coverage prerequisite on new or updated translation strings, so each deployment happens with whichever strings are available at that point, falling back to en.yml where necessary.

I've never looked to see what the average turnaround is between adding new strings (or updating existing ones) and how long until they are translated.

**Comment by gravitystorm** [comment]
> The CWG had developed [recommendations on changes to content and presentation](https://docs.google.com/document/d/1qthlWHa3-cQ9KcHai-TORAEnHD3LBI0vTi2OJ1TMz_k/edit), and that text has been ok'd by the LWG and Board.

This doesn't look finished to me, for example it has `[Support paragraph with donate button]` as the second paragraph.

> Would be good to have guidance from maintainers if this is simple enough that it is sufficient to make this change in those text files and submit as a PR, or require someone to get the website running locally to test? 

Sure, PRs are welcome. We do appreciate if the person making the PR also tests it locally, to make sure it works as intended. This reduces the burden on the reviewers.

**Comment by tomhughes** [comment]
Also while we know about the broken links we generally can't change them because they are part of third party mandated attributions - anything in the "our contributors" link is subject to agreement between LWG and upstream before we can accept changes.

**Comment by tomhughes** [comment]
I see that document also talks about a "sidebar" but I'm not sure that will be very consistent with the site design - it will depend on the details though.

**Comment by simonpoole** [comment]
> We make the changes to en.yml, the PR is merged, and then the translations come in at their own pace.

It needs to be pointed out that changing the text in the source file (en.yml) does not invalidate existing translations for a specific string. That is OK if it is a button or whatever. but particularly in the case of legal/semi-legal texts that is typically not what you want.

**Comment by simonpoole** [comment]
> Note there's a few other old issues which could possible be swept into these updates as well
...
> 
>     * [Redesign /copyright to unify the wiki Contributors page #1878](https://github.com/openstreetmap/openstreetmap-website/issues/1878)

This needs a technical solution, it is definitely not done with the rewording of a couple of paragraphs. 

See also https://github.com/openstreetmap/openstreetmap-website/pull/1480#issuecomment-391642084

**Comment by AntonKhorev** [comment]
> changing the text in the source file (en.yml) does not invalidate existing translations

If you want to force the text changes, you can rename the key, like this:

    important_legal_text: something

to

    important_legal_text_2024: something else

**Comment by mmd-osm** [comment]
To keep this issue actionable, I would suggest to skip the open points for the time being (that's: _support paragraph with donate button_, and the proposed new sidebar).  The intro text has a comment "should be styled as a call out with a great visual". I would also skip this for now, due to lack of relevant details.

These points can be covered in another issue once we get some more details from CWG.

Also, I'd suggest to skip #1878 for now, since it's a project on its own, really. If you believe a 3rd party source management system is needed, #1878 might be a better place to discuss some high level requirements. Depending on the scope/estimated effort, EWG involvement might be needed.

**Comment by mikelmaron** [comment]
@mmd-osm agree keep it actionable. some of these open topics could be refined quickly, once we have momentum of implementation. 

is there anyone willing and able to jump in an assist in implementation?
